README.md

```diff
@@ -24,7 +24,11 @@ Here is the expected format of an [`extensions.json`](./extensions.json) entry:
       // Unique Open VSX extension ID in the form "<namespace>.<name>"
       "id": "redhat.vscode-yaml",
       // Repository URL to clone and publish from
-      "repository": "https://github.com/redhat-developer/vscode-yaml"
+      "repository": "https://github.com/redhat-developer/vscode-yaml",
+      // (RECOMMENDED) The version to publish to Open VSX (defaults to the package.json version)
+      "version": "0.27.0",
+      // (RECOMMENDED) The Git branch, tag, or commit to check out before publishing (defaults to the repository's default branch)
+      "checkout": "v0.27.0",
```

Comment:
Since repositories don't always have dedicated "release" branches or tags, I prefer keeping `checkout` optional, and also keeping the first example minimal (i.e. only required parameters, since the optional ones are documented below).

```suggestion
      "repository": "https://github.com/redhat-developer/vscode-yaml"
```